app/controllers/podcast_episode_appearances_controller.rb

```diff
@@ -0,0 +1,114 @@
+class PodcastEpisodeAppearancesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create edit update destroy]
+  before_action :set_episode, only: %i[new]
+  before_action :set_podcast_episode_appearance, only: %i[show edit update destroy]
+  before_action :set_episode_with_strong_params, only: %i[create update]
+  before_action :set_podcast_episode_appearances, only: %i[index create update destroy]
+
+  def index; end
+
+  def show; end
+
+  def new
+    @podcast_episode_appearance = @episode.podcast_episode_appearances.build
+    authorize @podcast_episode_appearance
+  end
+
+  def edit
+    authorize @podcast_episode_appearance
+  end
+
+  def create
```

Comment:
The create and destroy methods can stay here I think, I'd imagine that the UI sends the parameters and gets JSON returned so that it can operate.